server/env

```diff
@@ -0,0 +1,8 @@
+AIRTABLE_API_KEY=
```

Comment:
Should these contents be in `.gitignore`? I think having an initial `server/env` might make sense, but we might not want to allow folks to commit changes later.

Reply:
++ on this

Reply:
The intent is to make the list of all the environment variables easily accessible for a developer to rename the file to `.env`, then fill out sensitive information and run locally on their machine.

Anything listed explicitly on that `server/env` is non-sensitive information.

Besides the listing these environment variables on a readme I'm not sure how else I would be able to easily communicate to a user/developer that they need them (unless they read the server file and the globals defined -- should that be the case?)

Reply:
I see - maybe we should leave a comment at the top of this file specifying that they shouldn't place their values in this file and move it instead? Alternatively, we can call this `server/env.example`.

Reply:
sounds good! I went with `server/env.example` let me know if there can be a better phrasing for the instructions.